react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -200,6 +201,13 @@ public void requestFocusFromJS() {
             @Override
             public void run() {
                 // let's pinpoint the caret line to ask the system to bring that line into the viewport
+                // we need to make sure that getLayout() isn't null
+                // because it will cause the NPE: https://github.com/wordpress-mobile/WordPress-Android/issues/11821
+                if (getLayout() == null) {
+                    AppLog.e(AppLog.T.EDITOR, "Layout is null");
```

Comment:
Since we bail on purpose, how about logging a warning instead of an error?

Also, I think the `Layout is null` offers too little context to anyone reading the app logs so, what do you think about using something like "Layout is null while trying to scroll inside a ReactAztecText block. Canceling the scroll."?

Reply:
> Since we bail on purpose, how about logging a warning instead of an error?
> 
> Also, I think the `Layout is null` offers too little context to anyone reading the app logs so, what do you think about using something like "Layout is null while trying to scroll inside a ReactAztecText block. Canceling the scroll."?

Thanks for the good feedback!
The log message should be better now.